test/test-compat.spec.js

```diff
@@ -2,15 +2,13 @@
 
 // tests mirrored in go-merkledag/pb/compat_test.go
 
-import chai from 'chai'
+import { assert } from 'aegir/chai'
 import { bytes } from 'multiformats'
 import { CID } from 'multiformats/cid'
-import { encode, decode } from '@ipld/dag-pb'
```

Comment:
what breaks for this to need to change? we could add back `"paths"` if it's typescript compiling that's failing; it's kind of nice to be consuming it as an external consumer would

Reply:
Honestly this paths thing is super complicated and I don't see other projects doing it that way. 😅

I think it's related to some of the issues we're seeing with duplicate multiformats causing errors.

From what I'm seeing, the `paths` thing is counter to how ESM works and doesn't play nice with esbuild.

I think the `ts-use` pattern could be useful for making sure it works as a consumer.